pkg/proto/datadog/api/v1/api.proto

```diff
@@ -98,6 +99,13 @@ service AgentSecure {
             body: "*"
         };
     };
+
+    rpc GetConfigFullState(google.protobuf.Empty) returns (datadog.config.GetStateConfigResponse) {
```

Comment:
Nit: drop the `Full` part, I don't think we have multiple level of state right ?